packages/contracts/src/LilVRGDA.sol

```diff
@@ -155,7 +161,16 @@ contract LilVRGDA is ILilVRGDA, LinearVRGDA, PausableUpgradeable, ReentrancyGuar
             "Invalid block number"
         );
 
-        uint256 _nextNounIdForCaller = nextNounIdForCaller();
+        // If going to mint Nouns that are founder rewards, increment the counts
+        uint256 _nextNounIdForCaller = nextNounId;
+        if (_nextNounIdForCaller <= 175300 && _nextNounIdForCaller % 10 == 0) {
+            _nextNounIdForCaller++;
+            lilNounderRewardNouns++;
+        }
+        if (_nextNounIdForCaller <= 175301 && _nextNounIdForCaller % 10 == 1) {
+            _nextNounIdForCaller++;
+            nounsDAORewardNouns++;
+        }
```

Comment:
Reward noun counts are updated before pricing while nextNounId is updated after which will cause a pricing mismatch when pulling the price for xxx2 nouns.